packages/commonwealth/client/scripts/views/components/react_quill_editor/react_quill_editor.tsx

```diff
@@ -194,6 +196,21 @@ const ReactQuillEditor = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [editorRef]);
 
+  useEffect(() => {
+    if (shouldFocus) {
+      // Important: We need to initially focus the editor and then focus it again after
+      // a small delay. Some code higher up in the tree will cause the quill
+      // editor to remount/redraw because of some force re-renders use emitters. This causes
+      // the editor to remain focused while losing the text cursor and not being able to type.
+      editorRef && editorRef.current && editorRef.current.focus();
+      setTimeout(
+        () => editorRef && editorRef.current && editorRef.current.focus(),
+        200
+      );
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
```

Comment:
@jnaviask To fix the RCA we have to prevent unnecessary quill editor re-renders and force renders from event emitters. That would be a big fix since it involves a lot of places, so for the time being I added this workaround.

Plz lmk if you think its not good.

Reply:
Don't love this solution but it'll work for now. @mzparacha can you open up a new ticket describing the specific force renders that the editor uses + a path to removing them?